src/overlays/actors/ovl_En_Bubble/z_en_bubble.h

```diff
@@ -10,7 +11,25 @@ typedef void (*EnBubbleActionFunc)(struct EnBubble*, PlayState*);
 typedef struct EnBubble {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnBubbleActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x110];
+    /* 0x148 */ ColliderJntSph colliderSphere;
+    /* 0x168 */ ColliderJntSphElement colliderElements[2];
+    /* 0x1F4 */ Vec3f unk1F4; // set but never used
+    /* 0x1F4 */ Vec3f unk1F8; // randomly generated, set but never used
+    /* 0x200 */ s16 unk200; // set to 8 when about to pop
+    /* 0x202 */ s16 explosionCountdown;
+    /* 0x204 */ char pad204[4]; // unused
+    /* 0x208 */ f32 graphicRotSpeed;
+    /* 0x20C */ f32 graphicEccentricity;
+    /* 0x210 */ f32 unk210; // set to 1.0f, never used
+    /* 0x214 */ f32 unk214; // set to 1.0f, never used
+    /* 0x218 */ f32 expansionWidth;
+    /* 0x21C */ f32 expansionHeight;
```

Comment:
I don't particularly like any of these four names. `eccentricity` is definitely wrong, that's `sqrt(1-b^2/a^2)` One of the terms here is more appropriate: https://en.wikipedia.org/wiki/Flattening

Secondly, we don't use `graphic` like this, I suppose it should be `model`.

Thirdly, the `expansion` names are misleading, it's really extra added onto the horizontal and the vertical radii; visually this applies *after* the two "graphic" variables have applied, so 3 stretches happen, one which rotates, then one horizontally and one vertically; of course there's no 3D since it's using the billboard matrix to make it face the camera the whole time.

Reply:
I can name it "Ellipticity' in your honor